vuepress/docs/next/tutorials/create/ms/update-data-model.md

```diff
@@ -9,22 +9,22 @@ sidebarDepth: 2
 This tutorial explains how to use the [Entando Component Generator](../../../docs/create/component-gen-overview.md) powered by [JHipster](https://www.jhipster.tech/) to quickly update the data model for your Entando project.
 
 ## Prerequisites
-Leverage an existing project or [create a project](./generate-microservices-and-micro-frontends.md) and navigate to the root folder. The steps below assume you're working in the top-level project directory.
+A [Blueprint-generated project](./generate-microservices-and-micro-frontends.md) 
 
 ## Tutorial
-1. Use JHipster to extract the current application description. The resulting JHIpster Domain Language (JDL) file contains your project's application configuration and entity definitions.
+The steps below assume you're working out of the root directory of the microservice, e.g. /microservices/conference-ms.
```

Comment:
root directory should be just /microservices or maybe it should read root directory of the conference microservice?

Reply:
actually if the MS is conference-ms then /microservices/conference-ms is the path to the root of the MS; same as your second rec but including the path